health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/VennDiagramChart.js

```
@@ -0,0 +1,271 @@
+  const { value } = useContext(FilterContext);
+  const { id, chartType } = data;
+  const tenantId = Digit?.ULBService?.getCurrentTenantId();
+  const [chartKey, setChartKey] = useState(id);
+
+  const { startDate, endDate, interval } = getInitialRange();
+  const { campaignId } = Digit.Hooks.useQueryParams();
+  // const { projectTypeId} = Digit.Hooks.useQueryParams();
+  // const selectedProjectTypeId = projectTypeId ? projectTypeId : Digit.SessionStorage.get("selectedProjectTypeId");
+  const today = new Date();
+  today.setHours(0, 0, 0, 0);
+  let todayDate = today;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'todayDate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)